dask_kubernetes/operator/kubecluster/kubecluster.py

```diff
@@ -867,7 +867,12 @@ def from_name(cls, name, **kwargs):
         --------
         >>> cluster = KubeCluster.from_name(name="simple-cluster")
         """
-        return cls(name=name, create_mode=CreateMode.CONNECT_ONLY, **kwargs)
+        return cls(
+            name=name,
+            create_mode=CreateMode.CONNECT_ONLY,
+            shutdown_on_close=False,
+            **kwargs,
+        )
```

Comment:
_Pseudo code and probably wont pass linting. But we could use the dict pipe merge now that 3.9 is our minimum!_

```suggestion
        defaults = {"create_mode": CreateMode.CONNECT_ONLY, "shutdown_on_close": False}
        kwargs = defaults | kwargs
        return cls(
            name=name,
            **kwargs,
        )
```